strapi/strapi
# feat(strapi): add experimental plugin:watch:link command to CLI

### What does it do?

* adds `strapi plugin:watch:link` experimental command to the CLI

### Why is it needed?

* Helps individuals connect their plugins to strapi apps either without using workspaces in the same project or if it's a separate package/project

### How to test it?

* use the experimental `0.0.0-experimental.81d72dd33765b4aa300a092f80857287eababa0a` version to create a plugin by running the command `npx @strapi/strapi@0.0.0-experimental.81d72dd33765b4aa300a092f80857287eababa0a plugin:init`
* replace the `@strapi/strapi` dependency version to match the experimental above
* run `yarn`
* run `yarn link-watch`
* follow the commands to link the plugin to the example app
* it should run & update

### Related issue(s)/PR(s)

* resolves CONTENT-1753
